include/boost/beast/core/string.hpp

```diff
@@ -33,6 +33,9 @@ iequals(
 /** A case-insensitive less predicate for strings.
 
     The case-comparison operation is defined only for low-ASCII characters.
+
+    As of c++14 containers using this class as the Compare type will take part
+    in heterogenious lookup if the search term is implicitly convertible to string_view.
```

Comment:
Spelling: heterogeneous
Capitalization: C++14
Punctuation: comma after C++14
"Compare" should be in back-quotes and string_view should be a `@ref`.

Reply:
For the error in spelling I am contrite. Unforgivable.

Reply:
Probably shouldn't capitalize heterogeneous

Reply:
I would say "key" instead of "search term".

Reply:
I see you are a proponent of the Oxford comma. The subject of bitter debates between literary experts in the UK :-)

Reply:
I could go either way on the comma